### DISCUSSION TITLE
可否实现多用户管理？例如输入不同的SS密码，对应不同的用户？

### DISCUSSION THREAD

**Discussion by hkix** [v2fly/v2ray-core#1453] [Q&A] [answered]
希望使用SS协议，看了文档没有提到多用户配置方式。请问是否支持多用户？

**Comment by kallydev** [comment, marked as answer]
Shadowsocks currently does not provide a single port multi-user feature.

- https://github.com/shadowsocks/shadowsocks-rust/issues/489
- https://github.com/shadowsocks/shadowsocks-org/issues/130
